Makefile

```diff
@@ -36,7 +36,7 @@ init-backend:
 test: test-backend # alias for quick access
 test-backend:
 	@cd ${RECIPLEASE_PATH} && \
-	${MVN} test
+	./scripts/test-backend.sh
```

Comment:
Normally this command is for running code related tests. i.e. for TDD. The checks for deployment should be in the actual deployment scripts.

Reply:
hmm but then it doesn't run for PRs